spec/rubocop/cop/layout/empty_lines_around_arguments_spec.rb

```diff
@@ -80,6 +80,52 @@ def anything; end
       expect(cop.messages)
         .to eq(['Empty line detected around arguments.'])
     end
+
+    context 'on correct lines' do
+      it 'for single offense example' do
```

Comment:
All of those should read like sentences - `it does this and that`. :-) Otherwise the RSpec reports look very confusing. You might want to consult other RuboCop spec files.